build_projects/dotnet-host-build/DebTargets.cs

```diff
@@ -131,6 +131,9 @@ public static BuildTargetResult GenerateSharedFrameworkDeb(BuildTargetContext c)
             var debFile = c.BuildContext.Get<string>("SharedFrameworkInstallerFile");
             var debianConfigFile = Path.Combine(Dirs.DebPackagingConfig, "dotnet-sharedframework-debian_config.json");
 
+            bool isDebian9 = c.BuildContext.Get<string>("TargetRID") == "debian.9-x64";
+            var libSslPackageName = isDebian9 ? "libssl1.0.2" : "libssl1.0.0";
```

Comment:
FYI - @bartonjs.  This is the version that we listed as a dependency for Debian 9 in the 2.0 branch:

https://github.com/dotnet/core-setup/blob/b64d29d85f553a85947586a6a4f0e7acee962a28/src/pkg/packaging/deb/dotnet-sharedframework-debian.9_config.json#L33

Just making sure you agree with this change, and the version number looks right to you.  The tests were failing to install the .deb package when it was pointing to `libssl1.0.0`.

Reply:
Yep, looks good.  That package name was introduced in deb9.  It'll be the same in deb10... probably.